test/integration/scenarios/stress/0.transfer_with_data.js.js

```diff
@@ -14,32 +14,31 @@
 
 'use strict';
 
-var Promise = require('bluebird');
-var lisk = require('lisk-elements').default;
-var accountFixtures = require('../../../fixtures/accounts');
-var randomUtil = require('../../../common/utils/random');
-var waitFor = require('../../../common/utils/wait_for');
-var sendTransactionsPromise = require('../../../common/helpers/api')
+const Promise = require('bluebird');
+const lisk = require('lisk-elements').default;
+const accountFixtures = require('../../../fixtures/accounts');
+const randomUtil = require('../../../common/utils/random');
+const waitFor = require('../../../common/utils/wait_for');
+const sendTransactionsPromise = require('../../../common/helpers/api')
 	.sendTransactionsPromise;
-var confirmTransactionsOnAllNodes = require('../common/stress')
+const confirmTransactionsOnAllNodes = require('../../utils/transactions')
 	.confirmTransactionsOnAllNodes;
 
+const broadcasting = process.env.BROADCASTING !== 'false';
```

Comment:
Consistency, in other test we're using `const BROADCASTING = process.env.BROADCASTING !== 'false';`